pkg/skaffold/kubernetes/portforward/kubectl_forwarder.go

```diff
@@ -127,20 +137,29 @@ func (k *KubectlForwarder) forward(parentCtx context.Context, pfe *portForwardEn
 	}
 }
 
-func portForwardCommand(ctx context.Context, k *kubectl.CLI, pfe *portForwardEntry, buf io.Writer) *kubectl.Cmd {
-	args := []string{
-		"--pod-running-timeout", "1s",
-		fmt.Sprintf("%s/%s", pfe.resource.Type, pfe.resource.Name),
-		fmt.Sprintf("%d:%d", pfe.localPort, pfe.resource.Port),
-		"--namespace", pfe.resource.Namespace,
+func portForwardArgs(ctx context.Context, pfe *portForwardEntry) []string {
+	args := []string{"--pod-running-timeout", "1s", "--namespace", pfe.resource.Namespace}
+
+	switch {
+	case pfe.resource.Type == "service" && os.Getenv("SKAFFOLD_DISABLE_SERVICE_FORWARDING") == "":
```

Comment:
the docs say that this var being set to "any value" will cause this to break out, but empty string does technically count as a value. I think you want to use `os.Lookupenv` here instead:

```suggestion
_, disableServiceForwarding := os.Lookupenv("SKAFFOLD_DISABLE_SERVICE_FORWARDING")
	switch {
	case pfe.resource.Type == "service" && !disableServiceForwarding:
```